docs/application/dotnet/guides/location-sensors/device-sensors.md

```diff
@@ -18,7 +18,8 @@ A device can have various physical and virtual sensors. The following table list
 
 | Supported sensor types                   |                                          |                                       |
 |----------------------------------------|----------------------------------------|-------------------------------------|
-| [Accelerometer](#accelerometer)          | [Heart Rate Monitor Sensor](#hrm)        | [Proximity Sensor](#proximity)        |
+| [Accelerometer](#accelerometer)          | [AutoRotation Sensor](#auto_rotation)    | [Proximity Sensor](#proximity)        |
+| [Heart Rate Monitor Sensor](#hrm)        | -                                        | -                                     |
```

Comment:
For this table, please list up the table items to follow the alphabetical order.
The alphabetical order is from top to button, left to right.
So, it should be:

| Supported sensor types                   |                                          |                                       |
|----------------------------------------|----------------------------------------|-------------------------------------|
| [Accelerometer](#accelerometer)          | [Humidity Sensor](#humidity)     | [Proximity Sensor](#proximity)        |
| [AutoRotation Sensor](#auto_rotation)        | [Light Sensor](#light)                                       | [Rotation Vector Sensor](#rotation)                                      |
| [Geomagnetic Rotation Vector Sensor](#mag_rotation) | [Linear Acceleration Sensor](#lin_accelerometer)              | [Sleep Monitor](#sleep_monitor)  |
| [Gravity Sensor](#gravity)               | [Magnetic Sensor](#magnetic)                   | [Temperature Sensor](#temperature)       |
| [Gyroscope](#gyro)                       | [Orientation Sensor](#orientation)  | [Ultraviolet Sensor](#ultraviolet)     |
| [Gyroscope Rotation Vector Sensor](#gyro_rotation) | [Pressure Sensor](#pressure)             | [Uncalibrated Gyroscope](#uncal_gyro)    |
| [Heart Rate Monitor Sensor](#hrm)             | [Pedometer](#pedometer)       | [Uncalibrated Magnetic Sensor](#uncal_magnetic) |

Reply:
@mijin85cho done!